githooks/hooks/trusted.go

```diff
@@ -238,7 +238,21 @@ func (t *ChecksumStore) Summary() string {
 
 // GetChecksumDirectoryGitDir gets the checksum file inside the Git directory.
 func GetChecksumDirectoryGitDir(gitDir string) string {
-	return path.Join(gitDir, ".githooks.checksums")
+	var conf string
+	gitx := git.NewCtxAt(gitDir)
+	scope := git.Traverse
+	conf, set := os.LookupEnv("GITHOOKS_CHECKSUMS_DIR")
+	if !set {
+		conf = gitx.GetConfig(GitCKChecksumsDir, scope)
```

Comment:
**discussion**: Can we make this setting an absolute path only.
That means you dont need a symlink to place in to the template directory. 
You can just set this path, a global absolute path (**also check it if it is absolute otherwise take the default route**)

**important**: The uninstall will always clean the local `.git/.githooks.checksums` in 
`cleanArtefactsInRepo` (it uses this function) but should not pick up any set `githooks.checkumsDir` settings or Env var. So i suggest to make `.githooks.checksums` a global string and just append it on `gitDir` in `cleanArtefactsInRepo`. 

**discussion:** The uninstaller should probably remove the associated `GITHOOKS_CHECKSUMS_DIR` or `GitCKChecksumsDir` if set in `runUninstallSteps`. Not sure. Might be a bit dangerous, but also senseful since you set it, then clean up properly.